modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js

```diff
@@ -67,6 +67,16 @@ AUI.add(
 		var TPL_REPEATABLE_ADD =
 			'<a class="icon-plus-sign lfr-ddm-repeatable-add-button" href="javascript:;"></a>';
 
+		var TPL_REPEATABLE_ICON =
```

Comment:
Don't inline icons, change this to

```suggestion
		var TPL_REPEATABLE_ICON = Liferay.Util.getLexiconIconTpl('drag');
```